Snakefile

```diff
@@ -489,29 +540,41 @@ if config["augmented_line_connection"].get("add_to_snakefile", False) == False:
 
     rule cluster_network:
         input:
-            network="networks/elec_s{simpl}.nc",
-            country_shapes="resources/shapes/country_shapes.geojson",
-            regions_onshore="resources/bus_regions/regions_onshore_elec_s{simpl}.geojson",
-            regions_offshore="resources/bus_regions/regions_offshore_elec_s{simpl}.geojson",
-            #gadm_shapes="resources/shapes/MAR2.geojson",
+            network="networks/" + RDIR + "elec_s{simpl}.nc",
+            country_shapes="resources/" + RDIR + "shapes/country_shapes.geojson",
+            regions_onshore="resources/"
+            + RDIR
+            + "bus_regions/regions_onshore_elec_s{simpl}.geojson",
+            regions_offshore="resources/"
+            + RDIR
+            + "bus_regions/regions_offshore_elec_s{simpl}.geojson",
+            #gadm_shapes="resources/" + RDIR + "shapes/MAR2.geojson",
             #using this line instead of the following will test updated gadm shapes for MA.
-            #To use: downlaod file from the google drive and place it in resources/shapes/
+            #To use: downlaod file from the google drive and place it in resources/" + RDIR + "shapes/
             #Link: https://drive.google.com/drive/u/1/folders/1dkW1wKBWvSY4i-XEuQFFBj242p0VdUlM
-            gadm_shapes="resources/shapes/gadm_shapes.geojson",
-            # busmap=ancient('resources/bus_regions/busmap_elec_s{simpl}.csv'),
+            gadm_shapes="resources/" + RDIR + "shapes/gadm_shapes.geojson",
+            # busmap=ancient('resources/" + RDIR + "bus_regions/busmap_elec_s{simpl}.csv'),
             # custom_busmap=("data/custom_busmap_elec_s{simpl}_{clusters}.csv"
             #                if config["enable"].get("custom_busmap", False) else []),
             tech_costs=COSTS,
         output:
-            network="networks/elec_s{simpl}_{clusters}.nc",
-            regions_onshore="resources/bus_regions/regions_onshore_elec_s{simpl}_{clusters}.geojson",
-            regions_offshore="resources/bus_regions/regions_offshore_elec_s{simpl}_{clusters}.geojson",
-            busmap="resources/bus_regions/busmap_elec_s{simpl}_{clusters}.csv",
-            linemap="resources/bus_regions/linemap_elec_s{simpl}_{clusters}.csv",
+            network="networks/" + RDIR + "elec_s{simpl}_{clusters}.nc",
+            regions_onshore="resources/"
+            + RDIR
+            + "bus_regions/regions_onshore_elec_s{simpl}_{clusters}.geojson",
+            regions_offshore="resources/"
+            + RDIR
+            + "bus_regions/regions_offshore_elec_s{simpl}_{clusters}.geojson",
+            busmap="resources/"
+            + RDIR
+            + "bus_regions/busmap_elec_s{simpl}_{clusters}.csv",
+            linemap="resources/"
+            + RDIR
+            + "bus_regions/linemap_elec_s{simpl}_{clusters}.csv",
```

Comment:
Any chance we can avoid the pre-commit changes?
Looks unnecessary. Not sure how they keep it unformatted in PyPSA-Eur.

Reply:
Otherwise, the PR looks good and can be merged. Little release_note might not hurt?

Reply:
I removed the pre-commit message and added the release note.

I checked the pre-commit and I don't see major differences. Not sure if the change is force-pushed.
Maybe they don't have such issue because their strings are shorted.
We added the subfolder bus_regions in this case.
I test few options and push changes if I see a nicer approach

Reply:
They have them as well: https://github.com/PyPSA/pypsa-eur/blob/b579f8c5ff55c83192e5fa9195e323a9a8384955/Snakefile#L467

In our case we have that more often because our paths are longer and we use more in the snakefile

Reply:
Ok! Let's merge if you give me a thumbs up :)